features/bootstrap/FeatureContext.php

```diff
@@ -13,6 +13,8 @@
 use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\CompositeLabel;
 use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\CompositeRelation;
 use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\Dummy;
+use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\EmbeddableDummy;
+use ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity\EmbeddedDummy;
```

Comment:
@teohhanhui @soyuka , just my opinion by working a little bit in functionnal test as a small contributor. It's really hard cause any change of existing tested entities, you have to fix a lots a functionnal tests (maybe due to comparaison with JSON should be equal to... and comparaison with entity field definition...) . An Issue about that would be good ^^